docs/dev/api/insights.md

```diff
@@ -3872,22 +5529,40 @@ Return information about builds and tests run included in the build. Also, provi
      <td><p><small>| QUERY | OPTIONAL | ARRAY OF STRINGS |</small></p><p>Limit results to only those run on the specified device.</p></td>
     </tr>
   </tbody>
+  <tbody>
+    <tr>
+     <td><code>device_group</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>Return results only for the specified device group. Available values are: <code>private</code>, <code>public</code>.</p></td>
```

Comment:
When there is more than one available value, we should use a bulleted list so the description is easier to read.
```suggestion
     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>Return results only for the specified device group. Available values are: <ul><li><code>private</code></li><li><code>public</code></li></ul></p></td>
```